.env-dist

```diff
@@ -41,19 +41,20 @@ BASKET_NEWSLETTER=mozilla-and-you
 # leave FXA_ENABLED empty to disable FXA
 FXA_ENABLED=
 FXA_SETTINGS_URL="https://stable.dev.lcip.org/settings"
-OAUTH_CLIENT_ID=cb1bef9d06bb9bc9
-OAUTH_CLIENT_SECRET=f5fb99de6e0af18ab17e013ac1d439903179a97a1c510fc10bc3bd50bbce089b
+OAUTH_CLIENT_ID=edd29a80019d61a1
+OAUTH_CLIENT_SECRET=a80feaad77c847275d39ac989ee12a873ef6b54cbc184128f86f2afecdf003b5
 OAUTH_AUTHORIZATION_URI="https://oauth-stable.dev.lcip.org/v1/authorization"
 OAUTH_PROFILE_URI="https://stable.dev.lcip.org/profile/v1/profile"
 OAUTH_TOKEN_URI="https://oauth-stable.dev.lcip.org/v1/token"
 
 # HIBP API for breach data
-HIBP_API_ROOT="https://stage.haveibeenpwned.com/api/v2/"
 # How many seconds to wait before refreshing upstream breach data from HIBP
 HIBP_RELOAD_BREACHES_TIMER=600
 # HIBP API for range search and subscription
 HIBP_KANON_API_ROOT="https://api.haveibeenpwned.com"
-HIBP_KANON_API_TOKEN=""
+HIBP_KANON_API_TOKEN=
+HIBP_API_ROOT="https://haveibeenpwned.com/api/v2"
+HIBP_API_TOKEN=
```

Comment:
Are these OK to be "naked", or should they be empty strings, a la `=""`?

Reply:
Yes, according to, https://www.npmjs.com/package/dotenv